### ISSUE TITLE
After I download DTU dataset, it can not run by the next guide.

### ISSUE THREAD

**Issue by ray2004a** [AaltoVision/relativeCameraPose#6]
Thanks for the sharing, and sorry for the problem as a newer of Torch.
I already downloaded full cleaned dataset on DTU(136GB) and extracted it to the "data" folder in roots of this git-master. 
Then run " th main.lua -do_evaluation -source_image_path data/Cleaned/ -weights ./pre-trained/siam_hybridnet_fullsized.t7", but it showed like below:

========================================================
invalid argument: -source_image_path	
Usage: [options]

Torch-7 relative pose estimation
Options:
 - GPU                   Default preferred GPU [1]
 - nGPU                  Number of GPUs to use by default [2]
 - manualSeed            Manually set RNG seed [333]
 - quaternions           Use quaternion parametrization. Rodriguez otherwise [true]
 - spp                   Use spatial pyramid pooling [true]
 - do_evaluation         Evaluation stage [false]
 - do_cropping           Crop the original images [false]
 - data_description_path Home directory for text data [./data]
 - source_img_path       Home directory for images of corresponding landmarks [./data/Cleaned]
 - image_size_x          Image width [1200]
 - image_size_y          Image width [1600]
 - crop_size             Crop size [227]
 - training_dataset_size Number of training pairs [39424]
 - test_dataset_size     Number of test pairs [35391]
 - learning_rate         Learning rate [0.001]
 - gamma                 inv learning rate decay type: lr * (1 + gamma * epoch) ^ (-power) [0.001]
 - power                 inv learning rate decay type: lr * (1 + gamma * epoch) ^ (-power) [0.5]
 - weight_decay          weight decay [1e-05]
 - beta1                 first moment coefficient (for adam solver) [0.9]
 - beta2                 second moment coefficient (for adam solver) [0.999]
 - train_batch_size      Training mini-batch size [22]
 - test_batch_size       Test (validation) mini-batch size [47]
 - epoch_number          Manual epoch number (useful on restarts) [1]
 - max_epoch             Number of training epochs [10]
 - snapshot_dir          Snapshot directory [./snapshots]
 - results_dir           Results directory [./results]
 - weights               Pretrained model to begin training from []
===============================================================

Could anyone face this problem before? It will be very thanked for any suggestion.

**Comment by hahahehe-coder** [comment]
Try to change the 19th line of opts.lua.
E.g:
`
cmd:option("-source_img_path", "/media/hahahehe/KESU/Cleaned",     'Home directory for images of corresponding landmarks')
`
No need to add -source_img_path parameters when you run the command.
`
th main.lua -do_evaluation -weights ./pre-trained/siam_hybridnet_fullsized.t7
`
